salt/states/saltmod.py

```diff
@@ -120,6 +121,9 @@ def state(
     pillar
         Pass the ``pillar`` kwarg through to the state function
 
+    pillarenv
+        The pillar environment to grab pillars from
+
```

Comment:
Please add a ``.. versionadded:: Nitrogen`` line here to specify when this argument was added. You can find other examples of this throughout the codebase if you're not sure how to add it. It should be indented at the same level as the "The pillar environment to grab pillars from" line.